src/main/java/io/dropwizard/primer/util/IPAddressUtil.java

```diff
@@ -0,0 +1,30 @@
+package io.dropwizard.primer.util;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * Created by pavan.kumar on 2019-09-24
+ */
+@Slf4j
+public class IPAddressUtil {
+
+    private static final String X_REAL_IP = "x-real-ip";
```

Comment:
We need to make this configurable. Default it to X-FORWARDED-FOR and anyone can override.